exchange/bidder.go

```diff
@@ -751,3 +744,40 @@ func hasShorterDurationThanTmax(ctx bidderTmaxContext, tmaxAdjustments TmaxAdjus
 	}
 	return false
 }
+
+func setRequestBody(req *adapters.RequestData, endpointCompression string) ([]byte, error) {
+	var requestBody []byte
+
+	switch strings.ToUpper(endpointCompression) {
```

Comment:
Hans shared an idea to normalize the casing on app startup to avoid the case difference. Otherwise, using `strings.EqualFold` will be faster without allocating memory.

Reply:
Test this and don't see a difference in allocations. Compiler might be optimizing it .